### DISCUSSION TITLE
How to update an app?

### DISCUSSION THREAD

**Discussion by brice-laurencin** [jbangdev/jbang#1636] [Q&A] [answered]
Very sorry if it was asked before, but the search did not help me on this one.

How do I update an app I installed? (here, quarkus)

Thanks!

**Comment by quintesse** [comment, marked as answer]
Hi @brice-laurencin , if you have a recent version of JBang applications should update automatically (although it can take up to 12 hours after a new version has been released before JBang detects it and updates). But if you really want to be sure you have the latest one you can run `jbang app install --fresh --force ....`. HTH

**Reply by koppor** [reply]
One needs to pay attention that the `-SNAPSHOT` dependencies are not updated automatically.

**Reply by maxandersen** [reply]
You shouldn't be using snapshot of you want stability :)

**Reply by koppor** [reply]
> You shouldn't be using snapshot of you want stability :)

I use JBang for rapid prototyping; until we get our native image running 😅. We even patch JBang scripts on the fly when the underlying SNAPSHOT would be updated: https://github.com/JabRef/jabref/blob/68ce7e48a6c0c358d465881bb0bcbc29bf34e970/.github/workflows/tests-code.yml#L555

So nice that JBang works so smoothly.

Only drawback is that the early adaptors of our CLI will run into issues... But we don't want to do weekly releases on Maven Central 😅
